src/compoennts/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Layout, Menu, theme } from "antd";
+import { CloseOutlined, } from '@ant-design/icons';
+import { ReactNode, useEffect, useState } from "react";
+import { menuItems } from "./data";
+import { Header } from "antd/es/layout/layout";
+import { cn } from "@/src/utils";
+import useWindowWidth from "@/src/hooks/useWindowWidth";
+
+
+const { Content, Sider } = Layout;
+
+
+export default function Sidebar({ children }: { children: ReactNode }) {
+    const [isCollapsed, setIsCollapsed] = useState<boolean>(true);
```

Comment:
@jairhuaman-applydigital Creo que podemos mejorar los estados al integrar un hook de posición.

https://dev.to/n8tb1t/tracking-scroll-position-with-react-hooks-3bbj

Reply:
No se necesita un hook de posicion, ant design ya lo hace por defecto, la variable isCollapsed es para controlar el sidebar desde el botón de Menu y close button, esto porque ant design provee un botón , pero  no es customizable.